crypto/ec/ecdsa_ossl.c

```diff
@@ -298,7 +298,7 @@ ECDSA_SIG *ossl_ecdsa_sign_sig(const unsigned char *dgst, int dgst_len,
         }
         if (BN_is_zero(s)) {
             /*
-             * if kinv and r have been supplied by the caller don't to
+             * if kinv and r have been supplied by the caller don't
```

Comment:
Not to be contrary, but wouldn't the flow of the message be better with a comma after "called"?

(I kinda sorta agree with @t-j-h, but only in so far that the sentence doesn't become confusing, or its flow diminished)

Reply:
ah okay, you mean a comma after "caller"?

Reply:
Argh, yes!  (early morning here, that'l be my excuse)